### PR TITLE
Issue #KN-848 feat: Scala version update release-5.5.0

### PR DESCRIPTION
**Pull request by ShouraySolanki**

### CODE REVIEW

File: platform-core/schema-validator/src/main/java/org/sunbird/schema/impl/JsonSchemaValidator.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ private void loadSchema() throws Exception {
             InputStream stream = new URL( basePath + "schema.json").openStream();
             this.schema = readSchema(stream);
         }else {
-            Path schemaPath = new File( basePath + "schema.json").toPath();
+            Path schemaPath = new File( basePath + "config.json").toPath();
```
Comment (maheshkumargangula) [inline platform-core/schema-validator/src/main/java/org/sunbird/schema/impl/JsonSchemaValidator.java:40]: Why did you change this?

**Review by maheshkumargangula — CHANGES_REQUESTED** [review]

**Comment by ShouraySolanki** [comment]
I will revert it back for content i didn't see the schema.json file so
changed it check if the path issue is due to this file. But still it was
failing.
